webpack/assets/javascripts/react_app/common/MountingService.js

```diff
@@ -17,6 +19,14 @@ export function mount(component, selector, data, flattenData = false) {
   }
 }
 
+export function umountAllComponents() {
+  let node = mountedNodes.shift();
+  while (node) {
+    ReactDOM.unmountComponentAtNode(node);
```

Comment:
I believe the webcomponents rendering is going to solve unmountig for us, we should just switch to that globally instead.